core/chains/evm/client/errors.go

```diff
@@ -350,54 +361,3 @@ func extractRPCError(baseErr error) (*JsonError, error) {
 	}
 	return &jErr, nil
 }
-
-// ExtractRevertReasonFromRPCError attempts to extract the revert reason from the response of
-// an RPC eth_call that reverted by parsing the message from the "data" field
-// ex:
-// kovan (parity)
-// { "error": { "code" : -32015, "data": "Reverted 0xABC123...", "message": "VM execution error." } } // revert reason always omitted
-// rinkeby / ropsten (geth)
-// { "error":  { "code": 3, "data": "0x0xABC123...", "message": "execution reverted: hello world" } } // revert reason included in message
-func ExtractRevertReasonFromRPCError(err error) (string, error) {
```

Comment:
background for removing this: extracting a string is not possible for custom errors (unknown ABI). I think its cleanest to simply use ExtractRPCError and just stringify it, then we don't lose any information and can support custom errors. Reverts are rare enough that its not such a big burden to just decode the hex from a revert log as needed when debugging.